corehq/apps/case_search/xpath_functions/comparison.py

```diff
@@ -1,12 +1,17 @@
+from datetime import datetime, timedelta
+
 from django.utils.translation import gettext
 from eulxml.xpath import serialize
 from eulxml.xpath.ast import Step
 
 from corehq.apps.case_search.dsl_utils import unwrap_value
-from corehq.apps.case_search.exceptions import CaseFilterError
+from corehq.apps.case_search.exceptions import CaseFilterError, XPathFunctionException
+from corehq.apps.case_search.xpath_functions.value_functions import _value_to_date
```

Comment:
nit: it's not the best practice to import private methods (starting with `_`). I think an argument can be made to refactor the name of this method to a public one `value_to_date`.